src/data/quality-assurance-quiz.ts

```diff
@@ -1,5 +1,15 @@
 const qualityAssuranceQuiz = [
-   {
+  {
+    Question: "What type of testing is done on a new build to ensure that new functionality has not broken previously tested functionality?",
```

Comment:
```suggestion
    Question: "What type of testing is done on a new feature to ensure that it doesn't accidentally break existing functionality?",
```

Reply:
Thank you for the suggestion on the question, it sounds a lot better that way. I will commit the suggestion now. 😊